tools/data/textrecog/textocr_converter.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+import math
+import os
+import os.path as osp
+from functools import partial
+
+import mmcv
+
+from mmocr.utils.fileio import list_to_file
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Generate training and validation set of TextOCR '
+        'by cropping box image.')
+    parser.add_argument('root_path', help='Root dir path of TextOCR')
+    parser.add_argument(
+        'n_proc', default=1, type=int, help='Number of processes to run')
+    args = parser.parse_args()
+    return args
+
+
+def process_img(args, src_image_root, dst_image_root):
+    # Dirty hack for multi-processing
+    img_idx, img_info, anns = args
+    src_img = mmcv.imread(osp.join(src_image_root, img_info['file_name']))
+    labels = []
+    for ann_idx, ann in enumerate(anns):
+        text_label = ann['utf8_string'].lower()
```

Comment:
Why use `.lower()` for text_label?

Reply:
I followed the implementation in `svt_converter.py` but it seems unnecessary